libbeat/docs/dashboards.asciidoc

```diff
@@ -102,6 +102,10 @@ predefined +{beatname_lc}-*+ index pattern and set it as the default.
 
 image:./images/kibana-created-indexes.png[Kibana configured indexes]
 
+NOTE: After importing the index-pattern in Kibana, you need to refresh each index-pattern in Kibana in order to update
```

Comment:
@ruflin The section about loading the dashboards comes after the user has data in Elasticsearch as the previous section is "Starting Packetbeat", if the user follows the Getting started guide step by step. This is tricky as anytime, new keys can appear after you refreshed already the index-pattern.
I think adding a question under FAQ is a good idea.